src/app/legacy/containers/SocialEmbed/index.test.jsx

```diff
@@ -39,17 +39,8 @@ describe('SocialEmbedContainer', () => {
       );
 
       expect(container.firstChild).toMatchSnapshot();
-      expect(
-        document.querySelector(
-          'head script[src="https://platform.twitter.com/widgets.js"]',
-        ),
-      ).toBeTruthy();
+      
       unmount();
-      expect(
-        document.querySelector(
-          'head script[src="https://platform.twitter.com/widgets.js"]',
-        ),
-      ).toBeFalsy();
```

Comment:
Do these need removed? Looks like if the fireEvent is added it should trigger rendering of the embed: https://github.com/bbc/simorgh/pull/10562/commits/fb200acefdc1feaee78f34ae0605536ed56c52c1